CHANGES.txt

```diff
@@ -10,6 +10,10 @@
 * Add `--exclude-editable` to ``pip freeze`` to exclude editable packages
   from installed package list.
 
+* Add `--no-progress <progress_bar>` to ``pip download`` and ``pip install``
+  to suppress progress bar in the console (:pull:`4194`)
```

Comment:
Good spotting!
Apologies for the mix-up and thanks for keeping this clean and tidy !